packages/core/package.json

```
@@ -59,7 +59,7 @@
     "markdown-it": "^12.3.2",
     "msgpackr": "^1.10.2",
     "p-debounce": "^2.1.0",
-    "perfect-scrollbar": "1.5.5",
+    "perfect-scrollbar": "^1.5.5",
```

```suggestion
    "perfect-scrollbar": "^1.3.0",
```

Should we maybe set this to the same version as it was before?

---

No, we're currently picking up 1.5.5 anyway and this is main, so no use having the old dependency going forward

---

But could it not be the case that adoptors resolve to 1.5.4 for a specific reason that we are not aware of? Then we are forcing them to update with the next release.
Just a thought though, not really a blocker from my POV

---

That's true, but we update versions all the time. Not doing that for fear of breaking client leads to standstill.